projects/schematics/src/shared/utils/file-utils_spec.ts

```diff
@@ -144,6 +147,24 @@ const VALID_TEST_CLASS = `
       }
     }
 `;
+const VALID_REMOVE_CONSTRUCTOR_PARAM_CLASS = `
+    import { Dummy } from '@angular/core';
```

Comment:
I didn't found any usage for those dummy lines. Maybe we can remove those? 3 appearances

Reply:
If you are talking about the `Dummy` import, I intentionally put it there, so that the import that we are actually interested in is not the first one. This tests if the import searching logic is working correctly.

Reply:
Oh okay, sounds logical